src/test/ui/issues/issue-34334.stderr

```diff
@@ -1,8 +1,8 @@
-error: expected one of `,` or `>`, found `=`
-  --> $DIR/issue-34334.rs:2:29
+error: Expected identifier
+  --> $DIR/issue-34334.rs:2:17
    |
 LL |     let sr: Vec<(u32, _, _) = vec![];
-   |         --                  ^ expected one of `,` or `>`
+   |         --      ^^^^^^^^^^^
```

Comment:
this diagnostic is somewhat unfortunate?

Reply:
Agree. Improving diagnostics will be done in a separate PR.